samples/SimpleDataSource/Tables/Metadata/FileStatsMetadataTable.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.Performance.SDK;
+using Microsoft.Performance.SDK.Processing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SampleCustomDataSource.Tables.Metadata
+{
+    //
+    // This is a sample Metadata table for .txt files
+    // Metadata tables are used to expose information about the data being processed, not the actual data being processed.
+    // Metadata would be something like "number of events in the file" or "file size" or any other number of things that describes the data being processed.
```

Comment:
1) Nit: change "would be something like" to "could be"

2) Use a comma and or instead of two ors:
of events in the file," "file size," or any other..